library/common/event/provisional_dispatcher.h

```diff
@@ -19,14 +19,13 @@ namespace Event {
  */
 class ProvisionalDispatcher : public Logger::Loggable<Logger::Id::main> {
 public:
-  ProvisionalDispatcher(Event::Dispatcher& event_dispatcher)
-      : event_dispatcher_(event_dispatcher) {}
+  ProvisionalDispatcher() {}
```

Comment:
```suggestion
  ProvisionalDispatcher() = default;
```

Reply:
I mean technically I can also leave it off altogether for the same effect, right?